cloudsql-postgresql-plugin/src/e2e-test/java/io/cdap/plugin/cloudsqlpostgresql/BQValidation.java

```diff
@@ -40,51 +42,52 @@
 public class BQValidation {
     static List<JsonObject> BigQueryResponse = new ArrayList<>();
     static List<Object> bigQueryRows = new ArrayList<>();
+    static JsonObject json;
 
```

Comment:
Didn't asked to create a JsonObject global . I asked to create this here. 
Gson gson = new Gson()
JsonObject json = gson.fromJson(----------);

Reply:
change done